templates/Xcode/TCA Observable Feature.xctemplate/___FILEBASENAME___.swift

```diff
@@ -0,0 +1,23 @@
+// MARK: - ___VARIABLE_featureName___
+@Reducer
+public struct ___VARIABLE_featureName___: Sendable, FeatureReducer {
+	@ObservableState
+	public struct State: Sendable, Hashable {
+		public init() {}
+	}
+
+	public typealias Action = FeatureAction<Self>
+
+	public enum ViewAction: Sendable, Equatable {
+		case appeared
+	}
+
+	public init() {}
+
```

Comment:
Should we not include the body here explicitly? I think @danvleju-rdx found out it was needed? Or I might misremember

Reply:
I've seen it as well, but I am able to make it compile without it. @danvleju-rdx, if it compiles without it does it mean we can drop it, or was it added for runtime behavior differences?

Reply:
just realised it was the second, so added it to template

Reply:
Yes, runtime issues, [see](https://github.com/radixdlt/babylon-wallet-ios/pull/1315)